counterparty-core/counterpartycore/lib/api/compose.py

```diff
@@ -420,15 +420,17 @@ def compose_fairminter(
     lock_quantity: bool = False,
     divisible: bool = True,
     description: str = "",
+    price: int = 0,
+    quantity_by_price: int = 1,
     **construct_params,
 ):
     """
     Composes a transaction to issue a new asset using the FairMinter protocol.
     :param address: The address that will be issuing the asset (e.g. $ADDRESS_1)
     :param asset: The asset to issue (e.g. MYASSET)
     :param asset_parent: The parent asset of the asset to issue
-    :param price: The price in XCP of the asset to issue (e.g. 10)
-    :param quantity_by_price: The quantity of asset to mint per `price` paid
+    :param lot_price: The price in XCP of the asset to issue (e.g. 10)
```

Comment:
let's just add a comment here indicating the old names

Reply:
done!